plugin-scalebar/src/main/java/com/mapbox/maps/plugin/scalebar/ScaleBarImpl.kt

```diff
@@ -312,20 +301,53 @@ class ScaleBarImpl : ScaleBar, View {
     canvas.drawText(text, x, y, textPaint)
   }
 
+  /**
+   * @return A tuple with the scale bar segment distance and the amount of bar segments
+   */
+  @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+  internal fun findSuitableScaleBarSegments(maxDistance: Float): Pair<Float, Int> {
```

Comment:
Is there any reason to return `Pair<Float, Int>` instead of `Pair<Int, Int>` here?

Reply:
Yes, we can now return values smaller than `1` in the case where `maxDistance` it is already smaller than `1`. So the scale bar will render a scale bar segment with decimal distance (see second screenshot in the PR description).